api/arc.yml

```diff
@@ -864,6 +866,13 @@ components:
         Whether we should skip fee validation or not.
       schema:
         type: boolean
+    fullStatusUpdates:
+      name: X-FullStatusUpdates
+      in: header
+      description: >-
+        Whether we should have full status updates in callback or not.
```

Comment:
We should add here that full status updates means an additional callback for status `SEEN_IN_ORPHAN_MEMPOOL` and `SEEN_ON_NETWORK`